synapse/lib/lmdbslab.py

```diff
@@ -605,21 +598,26 @@ async def syncLoopTask(clas):
 
     @classmethod
     async def syncLoopOnce(clas):
-        for slab in clas.syncset:
+        for slab in list(clas.allslabs.values()):
```

Comment:
``tuple()`` not ``list()`` here. the thing we're iterating over here is not gonna change.

Reply:
Believe it or not, constructing a list from a generator is measurably faster than constructing a tuple on recent versions of python.

Reply:
The argument for using list, is that, yes, it isn't going to change, but, constructing a list from an iterator is cheaper.